packages/dd-trace/src/plugins/log_plugin.js

```diff
@@ -3,6 +3,7 @@
 const { LOG } = require('../../../../ext/formats')
 const Plugin = require('./plugin')
 const { storage } = require('../../../datadog-core')
+const structuredLoggers = new Set(['bunyan', 'pino', 'winston'])
```

Comment:
A base class shouldn't know about its subclasses. I would say to either create a `StructuredLogPlugin` base class that these 3 can inherit from instead, or add a property like `this.structured = true` that could be added that the 3 classes could set.